backend/Dockerfile.worker

```diff
@@ -46,13 +48,14 @@ RUN export LD_LIBRARY_PATH=/opt/OpenBLAS/lib:$LD_LIBRARY_PATH
 
 # Install pe-source module
 # Sync the latest from cf-staging branch
-RUN git clone -b cf-source-staging https://github.com/cisagov/pe-reports.git && cd pe-reports && git checkout c9cbbd73b22ef38cabe1da6ba50aeb2dc0be4f99 && pip install . 
+RUN git clone -b cf-source-staging https://github.com/cisagov/pe-reports.git && cd pe-reports && git pull && pip install .
```

Comment:
We should still pin to a particular commit

Reply:
You are correct.  @colin-tim and I just had a conversation about this and will make that change on our respective PRs.